src/integer_mod_q/mat_polynomial_ring_zq/get.rs

```diff
@@ -356,6 +383,35 @@ mod test_mod {
     }
 }
 
+#[cfg(test)]
+mod test_get_mat {
+    use crate::{
+        integer::MatPolyOverZ,
+        integer_mod_q::{MatPolynomialRingZq, ModulusPolynomialRingZq},
+    };
+    use std::str::FromStr;
+
+    const LARGE_PRIME: u64 = u64::MAX - 58;
+
+    /// Ensure that the getter for a large modulus and large entries works.
+    #[test]
+    fn get_mat_large_entry_and_modulus() {
+        let modulus =
+            ModulusPolynomialRingZq::from_str(&format!("5  42 0 0 0 1 mod {LARGE_PRIME}")).unwrap();
+        let poly_mat = MatPolyOverZ::from_str("[[4  1 0 0 1, 1  42],[0, 1  -1]]").unwrap();
+        let matrix = MatPolynomialRingZq::from((&poly_mat, &modulus));
```

Comment:
We do not need a prime here, but you can leave it like that, if you want to.

Reply:
i will leave it as it is